Resources/Textures/Mobs/Animals/nymph.rsi/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "#DionaNotAWood",
```

Comment:
if these are from bay, then you need to use the license that used on bay. A look on the repo says that the assets are licensed under the `CC-BY-SA-3.0` license.